pkg/reconciler/contour/config/contour.go

```diff
@@ -49,18 +51,29 @@ type Contour struct {
 	DefaultTLSSecret      *types.NamespacedName
 	TimeoutPolicyResponse string
 	TimeoutPolicyIdle     string
+	CORSPolicy            *CORSPolicy
+}
+
+type CORSPolicy struct {
```

Comment:
question:

what do you think importing Contour's definition of the CORS section of the HTTPProxy?

I think that is what you marshal it into anyway later on in `pkg/reconciler/contour/resources/httpproxy.go` on line 307? 

https://github.com/projectcontour/contour/blob/main/apis/projectcontour/v1/httpproxy.go#L494

Reply:
oh, didn't see Dave's comments, he said the same thing